### PR TITLE
PR4_changes03_on3_3_24

### PR DESCRIPTION
**Pull request by sheppardw**

### CODE REVIEW

**Comment by sheppardw** [comment]
Please ignore "PR4_changes03_on3_3_24". This is old.

The other 2 are ready for review and should have been on separate branches. sorry!

File: source/04-PR/03.ptx [hunk 1]
```diff
@@ -11,14 +11,14 @@
     </objectives>
     <subsection>
     <title>Activities</title>
-    <activity xml:id="activity-3-1">
+    <activity xml:id="Long Division Basic">
```
Comment (siwelwerd) [inline source/04-PR/03.ptx:14]: Note that spaces are not allowed in `xml:id`s.  Use a - to separate words.  I will fix all these shortly

File: source/04-PR/04.ptx [hunk 2]
```diff
@@ -10,22 +10,185 @@
...
-          <p> A number c is a zero of a polynomial function f(x) if and only if 
-             x-c  is a factor of f(x).</p>
+          <p> A number <m>c</m> is a zero of a polynomial function <m>f(x)</m> if and only if 
+             <m>x-c</m>  is a factor of <m>f(x)</m>.</p>
         </statement>
       </theorem>
+
+      <activity xml:id="zeros-cubic-factoring">
+        <introduction>
+          <task><p>Find the zeros of the polynomial by factoring <m>f(x)=x^3-x^2-56x</m>.
```
Comment (siwelwerd) [inline source/04-PR/04.ptx:59]: Your `<task>`s should follow an `<introduction>`, not be nested inside.  For something like this with only one real task, you can instead just use a `<statement>` without an `<introduction>`.

File: source/04-PR/04.ptx [hunk 3]
```diff
@@ -10,22 +10,185 @@
...
+            <li> <m>3</m> and <m>5</m></li>
+            </ol>
+        </p>
+        </task>
+        <task><p> How do the <m>x</m> intercepts relate to the factored form of the function?
+        </p></task>
+      
+      </activity> 
+      <definition xml:id="Real-zeros"> 
+        <statement> <p>Real zeros of a polynomial function are the same as the <m>x</m> intercepts.
```
Comment (siwelwerd) [inline source/04-PR/04.ptx:44]: Use a `<term>` here to make clear what you are defining

**Review by siwelwerd — APPROVED** [review]

**Pull request merged**